Backend/dev/migrations/dynamic-offer-driver-app/0133-added-driver-speed-configs.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE atlas_driver_offer_bpp.driver_intelligent_pool_config
+ADD COLUMN speed_normalizer DOUBLE PRECISION DEFAULT 28,
+ADD COLUMN speed_weightage_of_driver INT DEFAULT 5,
```

Comment:
not tested in local. As column names are not matching with names in Persistent Table